packages/apps/spaces/app/src/components/EmailMultiCreatableSelect/MultiValueWithActionMenu.tsx

```diff
@@ -190,26 +193,22 @@ export const MultiValueWithActionMenu: FC<MultiValueWithActionMenuProps> = ({
             },
           }}
         >
-          <chakraComponents.MultiValue {...rest}>
+          <chakraComponents.MultiValue
```

Comment:
This component should use the new Select implemented with tw.

Reply:
I will create separate task for that

Reply:
Actually, why is chakraComponents.MultiValue which is a inner component from chakra-react-select used inside a Tooltip? And the same goes for MenuItems. Menu items shoudl be used along with Menu.

Reply:
we are rendering email validation inside tooltip cc. @steynviljoen67